generic/tests/cfg/timedrift.cfg

```diff
@@ -35,21 +38,27 @@
                     drift_threshold = 10
                     drift_threshold_single = 3
                 - with_stop:
-                    # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-                    # and the time drift is seen because "stop" monitor command also stops PM timer,
-                    # so this is unsupported scenario from Win2012.
-                    Windows:
-                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                     type = timedrift_with_stop
                     stop_interations = 1
                     stop_time = 60
                     drift_threshold = 10
                     drift_threshold_single = 3
                     variants:
                         - clock_host:
+                            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
+                            # and the time drift is seen because "stop" monitor command also stops PM timer,
+                            # so this is unsupported scenario from Win2012.
+                            Windows:
+                                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                             rtc_clock = host
                         - clock_vm:
                             rtc_clock = vm
+                    variants:
+                        - base_utc:
+                            only Linux
+                        - base_localtime:
+                            only Windows
+                        - base_date:                        
```

Comment:
Yes.  For "base_utc", "rtc_base = utc" as defined in qemu_vm.py, for "base_localtime", "rtc_base = localtime" as defined in windows.cfg.  
For "base_date", "rtc_base = 2006-6-17" is defined in Windows.cfg as this patch https://github.com/avocado-framework/avocado-vt/pull/1073